static/settings.js

```diff
@@ -186,6 +186,13 @@ function setupSettings(root, settings, onChange, langId) {
 
     add(root.find('.newEditorLastLang'), 'newEditorLastLang', true, Checkbox);
 
+    var formats = ["Google", "LLVM", "Mozilla", "Chromium", "WebKit"];
```

Comment:
Would it be a lot of hassle to put this in an API call so we always have 1 list of these options server- and client-side?

Reply:
This is an excelent idea, but sadly I think we might want to hold on to that until someone wants to implement a different tool, not for how hard it is, but admittedly, for how lazy I am! (Sorry!)